dist/aframe-altspace-component.js

```diff
@@ -250,6 +255,33 @@
 	  }
 	});
 
+	(function(){
+
+	  function setColliderFlag(obj, state) {
+	    obj.userData.altspace = {collider: {enabled: state}};
+	    obj.traverse(function (obj) {
+	      if (obj instanceof THREE.Mesh) {
+	        obj.userData.altspace = {collider: {enabled: state}};
+	      }
+	    })
+	  }
+
+	  AFRAME.registerComponent('altspace-collider', {
+	    schema: { enabled: { default: true } },
+	    init: function () {
+	      setColliderFlag(this.el.object3D, this.data.enabled);
+	      this.el.addEventListener('model-loaded', function(){
+	        setColliderFlag(this.el.object3D, this.data.enabled);
```

Comment:
I'm not entirely sure if `this` is bound to the component here. You may want to add a `bind(this)` at the end of this callback function